docs/realm.js

```diff
@@ -288,13 +288,44 @@ Realm.defaultPath;
  *   object types in this Realm. **Required** when first creating a Realm at this `path`.
  * @property {number} [schemaVersion] - **Required** (and must be incremented) after
  *   changing the `schema`.
- * @property {Object} [sync] - Sync configuration parameters with the following 
+ * @property {Object} [sync] - Sync configuration parameters with the following
  *   child properties:
  *   - `user` - A `User` object obtained by calling `Realm.Sync.User.login`
  *   - `url` - A `string` which contains a valid Realm Sync url
  *   - `error` - A callback function which is called in error situations
  *   - `validate_ssl` - Indicating if SSL certificates must be validated
  *   - `ssl_trust_certificate_path` - A path where to find trusted SSL certificates
+ *   - `ssl_verify_callback` - A callback function used to accept or reject the server's
```

Comment:
why not call this open_ssl_verify_callback if this is respected only for open_ssl systems.

Reply:
Good idea. I will do that.

The idea was that it might be implemented for other crypto libraries in the future, but
that is probably not going to happen soon. I will rename it.